### DISCUSSION TITLE
How to save all parsed contests/problems in one folder?

### DISCUSSION THREAD

**Discussion by RitikGandhi21** [cpeditor/cpeditor#1044] [Q&A] [answered]
How can I save all the codes in one folder whenever I parse a problem?
I am unable to locate where my codes are saved and also is there a way to rename the problem because it just shows A,B,C,etc and instead can it show the contest number or the full problem name?

Whenever I parse a problem I want it's code to be saved by default on my system in a particular folder with the full problem name or contestnumber_A/B/C.cpp something like this?

Thank you

**Comment by ouuan** [comment, marked as answer]
You can set [Default File Path For Problem URLs](https://cpeditor.org/docs/preferences/file-path/#default-file-path-for-problem-urls).
